backend/src/cache/deployer/deploy-cache-service.sh

```diff
@@ -24,6 +24,17 @@ echo "Start deploying cache service to existing cluster:"
 
 NAMESPACE=${NAMESPACE_TO_WATCH:-kubeflow}
 mutatingWebhookConfigName="cache-webhook"
+
+# Should fail if there are connectivity problems
+# Gotcha: Listing all objects requires list permission, but when listing a single oblect kubecttl will fail if it's not found unless --ignore-not-found is specified.
+kubectl get mutatingwebhookconfigurations "${mutatingWebhookConfigName}" --namespace "${NAMESPACE}" --ignore-not-found >webhooks.txt
```

Comment:
Can we also rename as `MUTATING_WEBHOOK_CONFIGURATIONS` for a consistent naming convention? Up to you

Reply:
Done.